### PR TITLE
Currency: Added currency

### PR DESCRIPTION
**Pull request by Roverr**
Hey,

Unfortunately I didn't find the random currency feature which I had had to use, so therefore I ended up implementing it. Hope you find it useful!

I wanted to create this PR to the develop branch, but it's behind the master. Let me now if I should change it.

I didn't want to overwrite your `BENCHMARKS.md`, but here are the results:
```
BenchmarkAddress-8              	  500000	      2552 ns/op	     255 B/op	       7 allocs/op
BenchmarkStreet-8               	 2000000	       965 ns/op	      62 B/op	       3 allocs/op
BenchmarkStreetNumber-8         	 3000000	       445 ns/op	      36 B/op	       2 allocs/op
BenchmarkStreetPrefix-8         	10000000	       222 ns/op	       0 B/op	       0 allocs/op
BenchmarkStreetName-8           	10000000	       170 ns/op	       0 B/op	       0 allocs/op
BenchmarkStreetSuffix-8         	10000000	       175 ns/op	       0 B/op	       0 allocs/op
BenchmarkCity-8                 	 3000000	       423 ns/op	      15 B/op	       1 allocs/op
BenchmarkState-8                	10000000	       167 ns/op	       0 B/op	       0 allocs/op
BenchmarkStateAbr-8             	10000000	       156 ns/op	       0 B/op	       0 allocs/op
BenchmarkZip-8                  	 3000000	       525 ns/op	       9 B/op	       1 allocs/op
BenchmarkCountry-8              	10000000	       196 ns/op	       0 B/op	       0 allocs/op
BenchmarkLatitude-8             	50000000	        27.8 ns/op	       0 B/op	       0 allocs/op
BenchmarkLongitude-8            	50000000	        28.8 ns/op	       0 B/op	       0 allocs/op
BenchmarkBeerName-8             	10000000	       132 ns/op	       0 B/op	       0 allocs/op
BenchmarkBeerStyle-8            	10000000	       169 ns/op	       0 B/op	       0 allocs/op
BenchmarkBeerHop-8              	10000000	       142 ns/op	       0 B/op	       0 allocs/op
BenchmarkBeerYeast-8            	10000000	       140 ns/op	       0 B/op	       0 allocs/op
BenchmarkBeerMalt-8             	10000000	       148 ns/op	       0 B/op	       0 allocs/op
BenchmarkBeerIbu-8              	10000000	       118 ns/op	       8 B/op	       2 allocs/op
BenchmarkBeerAlcohol-8          	 5000000	       358 ns/op	      40 B/op	       3 allocs/op
BenchmarkBeerBlg-8              	 5000000	       361 ns/op	      48 B/op	       3 allocs/op
BenchmarkBool-8                 	50000000	        38.6 ns/op	       0 B/op	       0 allocs/op
BenchmarkColor-8                	10000000	       139 ns/op	       0 B/op	       0 allocs/op
BenchmarkSafeColor-8            	10000000	       124 ns/op	       0 B/op	       0 allocs/op
BenchmarkHexColor-8             	 2000000	       617 ns/op	      24 B/op	       3 allocs/op
BenchmarkRGBColor-8             	10000000	       119 ns/op	      32 B/op	       1 allocs/op
BenchmarkCompany-8              	 3000000	       491 ns/op	      22 B/op	       1 allocs/op
BenchmarkCompanySuffix-8        	10000000	       132 ns/op	       0 B/op	       0 allocs/op
BenchmarkBuzzWord-8             	10000000	       147 ns/op	       0 B/op	       0 allocs/op
BenchmarkBS-8                   	10000000	       149 ns/op	       0 B/op	       0 allocs/op
BenchmarkContact-8              	 1000000	      1779 ns/op	     178 B/op	       7 allocs/op
BenchmarkPhone-8                	 2000000	       547 ns/op	      16 B/op	       1 allocs/op
BenchmarkEmail-8                	 1000000	      1071 ns/op	     130 B/op	       5 allocs/op
BenchmarkCurrency-8             	10000000	       156 ns/op	      32 B/op	       1 allocs/op
BenchmarkCurrencyShort-8        	10000000	       133 ns/op	       0 B/op	       0 allocs/op
BenchmarkCurrencyLong-8         	10000000	       138 ns/op	       0 B/op	       0 allocs/op
BenchmarkDate-8                 	 5000000	       377 ns/op	       0 B/op	       0 allocs/op
BenchmarkDateRange-8            	10000000	       188 ns/op	       0 B/op	       0 allocs/op
BenchmarkMonth-8                	30000000	        49.2 ns/op	       0 B/op	       0 allocs/op
BenchmarkDay-8                  	30000000	        39.7 ns/op	       0 B/op	       0 allocs/op
BenchmarkYear-8                 	20000000	        80.7 ns/op	       0 B/op	       0 allocs/op
BenchmarkHour-8                 	50000000	        42.7 ns/op	       0 B/op	       0 allocs/op
BenchmarkMinute-8               	30000000	        41.1 ns/op	       0 B/op	       0 allocs/op
BenchmarkSecond-8               	50000000	        39.9 ns/op	       0 B/op	       0 allocs/op
BenchmarkNanoSecond-8           	50000000	        41.2 ns/op	       0 B/op	       0 allocs/op
BenchmarkMimeType-8             	10000000	       125 ns/op	       0 B/op	       0 allocs/op
BenchmarkExtension-8            	10000000	       138 ns/op	       0 B/op	       0 allocs/op
BenchmarkGenerate-8             	 1000000	      2022 ns/op	     414 B/op	      11 allocs/op
BenchmarkHackerPhrase-8         	  200000	      6586 ns/op	    2293 B/op	      26 allocs/op
BenchmarkHackerAbbreviation-8   	10000000	       137 ns/op	       0 B/op	       0 allocs/op
BenchmarkHackerAdjective-8      	10000000	       136 ns/op	       0 B/op	       0 allocs/op
BenchmarkHackerNoun-8           	10000000	       151 ns/op	       0 B/op	       0 allocs/op
BenchmarkHackerVerb-8           	10000000	       220 ns/op	       0 B/op	       0 allocs/op
BenchmarkHackerIngverb-8        	10000000	       170 ns/op	       0 B/op	       0 allocs/op
BenchmarkHipsterWord-8          	10000000	       133 ns/op	       0 B/op	       0 allocs/op
BenchmarkHipsterSentence-8      	 1000000	      2372 ns/op	     782 B/op	       9 allocs/op
BenchmarkHipsterParagraph-8     	   30000	     41740 ns/op	   18499 B/op	     170 allocs/op
BenchmarkImageURL-8             	10000000	       143 ns/op	      38 B/op	       3 allocs/op
BenchmarkDomainName-8           	 2000000	       826 ns/op	      76 B/op	       3 allocs/op
BenchmarkDomainSuffix-8         	10000000	       205 ns/op	       0 B/op	       0 allocs/op
BenchmarkURL-8                  	 1000000	      2036 ns/op	     279 B/op	       8 allocs/op
BenchmarkHTTPMethod-8           	10000000	       145 ns/op	       0 B/op	       0 allocs/op
BenchmarkIPv4Address-8          	 3000000	       501 ns/op	      48 B/op	       5 allocs/op
BenchmarkIPv6Address-8          	 2000000	       850 ns/op	      96 B/op	       7 allocs/op
BenchmarkUsername-8             	 3000000	       421 ns/op	      16 B/op	       2 allocs/op
BenchmarkPassword-8             	 1000000	      1868 ns/op	     432 B/op	       7 allocs/op
BenchmarkJob-8                  	 1000000	      1090 ns/op	      86 B/op	       2 allocs/op
BenchmarkJobTitle-8             	10000000	       121 ns/op	       0 B/op	       0 allocs/op
BenchmarkJobDescriptor-8        	10000000	       160 ns/op	       0 B/op	       0 allocs/op
BenchmarkJobLevel-8             	10000000	       150 ns/op	       0 B/op	       0 allocs/op
BenchmarkName-8                 	 3000000	       497 ns/op	      17 B/op	       1 allocs/op
BenchmarkFirstName-8            	 5000000	       216 ns/op	       0 B/op	       0 allocs/op
BenchmarkLastName-8             	10000000	       151 ns/op	       0 B/op	       0 allocs/op
BenchmarkNamePrefix-8           	10000000	       118 ns/op	       0 B/op	       0 allocs/op
BenchmarkNameSuffix-8           	10000000	       136 ns/op	       0 B/op	       0 allocs/op
BenchmarkNumber-8               	50000000	        35.1 ns/op	       0 B/op	       0 allocs/op
BenchmarkNumerify-8             	 3000000	       408 ns/op	      16 B/op	       1 allocs/op
BenchmarkShuffleInts-8          	 5000000	       259 ns/op	       0 B/op	       0 allocs/op
BenchmarkCreditCard-8           	 1000000	      1259 ns/op	      96 B/op	       6 allocs/op
BenchmarkCreditCardType-8       	10000000	       122 ns/op	       0 B/op	       0 allocs/op
BenchmarkCreditCardNumber-8     	 2000000	       671 ns/op	      16 B/op	       1 allocs/op
BenchmarkCreditCardExp-8        	10000000	       220 ns/op	       8 B/op	       3 allocs/op
BenchmarkCreditCardCvv-8        	10000000	       169 ns/op	       3 B/op	       1 allocs/op
BenchmarkSSN-8                  	 3000000	       413 ns/op	      16 B/op	       1 allocs/op
BenchmarkGender-8               	30000000	        41.0 ns/op	       0 B/op	       0 allocs/op
BenchmarkPerson-8               	  200000	      7427 ns/op	     823 B/op	      28 allocs/op
BenchmarkSimpleStatusCode-8     	20000000	        81.5 ns/op	       0 B/op	       0 allocs/op
BenchmarkStatusCode-8           	20000000	        85.7 ns/op	       0 B/op	       0 allocs/op
BenchmarkLetter-8               	20000000	        68.2 ns/op	       4 B/op	       1 allocs/op
BenchmarkLexify-8               	 5000000	       264 ns/op	       8 B/op	       1 allocs/op
BenchmarkShuffleStrings-8       	 5000000	       273 ns/op	       0 B/op	       0 allocs/op
BenchmarkUUID-8                 	10000000	       126 ns/op	      48 B/op	       1 allocs/op
BenchmarkUserAgent-8            	 1000000	      1558 ns/op	     313 B/op	      12 allocs/op
BenchmarkChromeUserAgent-8      	 1000000	      1173 ns/op	     193 B/op	       9 allocs/op
BenchmarkFirefoxUserAgent-8     	 1000000	      1888 ns/op	     389 B/op	      13 allocs/op
BenchmarkSafariUserAgent-8      	 1000000	      1905 ns/op	     563 B/op	      18 allocs/op
BenchmarkOperaUserAgent-8       	 1000000	      1289 ns/op	     227 B/op	      11 allocs/op
BenchmarkWord-8                 	20000000	       118 ns/op	       0 B/op	       0 allocs/op
BenchmarkSentence-8             	 1000000	      2132 ns/op	     721 B/op	      10 allocs/op
BenchmarkParagraph-8            	   50000	     38584 ns/op	   16106 B/op	     172 allocs/op
PASS
```

PS: Values are copied from ChanceJS Source
Also thanks for the great lib! 😄

### CODE REVIEW

File: currency.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+)
+
+// CurrencyInfo is a struct of currency information
+type CurrencyInfo struct {
+	Code string
+	Name string
+}
+
+// Currency will generate a struct with random currency information
+func Currency() *CurrencyInfo {
```
Comment (brianvoe) [inline currency.go]: Can we put the items of currency full in here and just get rid of CurrencyFull. Just a little redundent.

File: currency.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+// CurrencyLong will generate a random long currency name
+func CurrencyLong() string {
+	return getRandValue([]string{"currency", "long"})
+}
+
+// CurrencyFull gives back a full currency with code and name
+func CurrencyFull() *CurrencyInfo {
+	index := rand.Intn(len(data.Data["currency"]["short"]))
+	return &CurrencyInfo{
+		Code: data.Data["currency"]["short"][index],
```
Comment (brianvoe) [inline currency.go]: Maybe mimic the naming convention:
Code = Short
Name = Long

File: currency_test.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package gofakeit
+
+import (
+	"fmt"
+	"testing"
+)
+
+func ExampleCurrency() {
+	Seed(20)
```
Comment (brianvoe) [inline currency_test.go]: I know it seems silly but ive been using my lucky number 11 as the seed

File: currency_test.go [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+func BenchmarkCurrency(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		Currency()
+	}
+}
+
+func ExampleCurrencyShort() {
+	Seed(20)
```
Comment (brianvoe) [inline currency_test.go]: Same

File: currency_test.go [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+func BenchmarkCurrencyShort(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		CurrencyShort()
+	}
+}
+
+func ExampleCurrencyLong() {
+	Seed(20)
```
Comment (brianvoe) [inline currency_test.go]: Same

**Comment by brianvoe** [comment]
This looks good! Nice Job! Just a couple of changes and see if you can get codeclimate to be happy and ill merge this bad boy in!

Thanks!

**Comment by brianvoe** [comment]
P.S. If you want to add those couple of lines into the benchmark you are more than welcome to. No need to override the whole thing.

**Comment by Roverr** [comment]
Okay I think the issues you mentioned should be fixed now! Let me know what you think!

File: currency_test.go [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+func BenchmarkCurrency(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		Currency()
+	}
+}
+
+func ExampleCurrencyShort() {
+	Seed(11)
+	fmt.Println(CurrencyShort())
```
Comment (brianvoe) [inline currency_test.go]: Same

File: currency_test.go [hunk 7]
```diff
@@ -0,0 +1,39 @@
+package gofakeit
+
+import (
+	"fmt"
+	"testing"
+)
+
+func ExampleCurrency() {
+	Seed(11)
+	fmt.Println(Currency())
```
Comment (brianvoe) [inline currency_test.go]: // Output: the value that gets outputted

File: currency_test.go [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+func BenchmarkCurrencyShort(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		CurrencyShort()
+	}
+}
+
+func ExampleCurrencyLong() {
+	Seed(11)
+	fmt.Println(CurrencyLong())
```
Comment (brianvoe) [inline currency_test.go]: Same

**Comment by brianvoe** [comment]
Perfect! Thanks for this PR!

**Comment by brianvoe** [comment]
v3.1.0

**Pull request merged**